policy/LsstSimMapper.paf

```diff
@@ -672,378 +532,46 @@ datasets: {
         storage:    "FitsCatalogStorage"
         tables:        raw
     }
+    # dataset defined in obs_base modified in this mapper as follows:
     deepCoadd_forced_src: {
         template:      "deepCoadd-results/%(filter)s/%(tract)d/%(patch)s/forced-%(filter)s-%(tract)d-%(patch)s.fits"
-        python:        "lsst.afw.table.SourceCatalog"
-        persistable:        "ignored"
-        storage:    "FitsCatalogStorage"
     }
+    # dataset defined in obs_base modified in this mapper as follows:
     deepCoadd_forced_config: {
         template:      "config/deepCoadd_forced.py"
         python:        "lsst.meas.base.forcedCoadd.ForcedCoaddMeasurementConfig"
```

Comment:
I think the `python` entry here must either be incorrect or the `obs_base` one is.  Probably the former.